codex/utils/stintutils.nim

```diff
@@ -0,0 +1,4 @@
+import pkg/stint
+
+func fromString*(T: typedesc[StUint|StInt], s: string): T {.inline.} =
```

Comment:
I would suggest to call this `fromDecimals()`, to be more explicit. And it matches the `fromHex()` naming convention.

Reply:
Good idea, thanks! 👍